docs/operator-docs/bulk-operations.md

```diff
@@ -50,7 +50,7 @@ Bulk operations are generally submitted via the endpoint for the relevant data t
 
     ![Screenshot of Cumulus dashboard showing modal window for triggering bulk granule operations with query, index, and workflow information populated](assets/bulk-granules-query-2.png)
 
-9. Click the "Run Bulk Operations" button. You should see a confirmation message, including an ID for the async operation that was started to handle your bulk action. You can [track the status of this async operation on the Operations dashboard page](#status-tracking), which can be visited by click the "Go To Operations" button:
+9. Click the "Run Bulk Operations" button. You should see a confirmation message, including an ID for the async operation that was started to handle your bulk action. You can [track the status of this async operation on the Operations dashboard page](#status-tracking), which can be visited by clicking the "Go To Operations" button:
```

Comment:
Nit:   Is that dialog box *really* that tall?    If that's just a rendering/resolution artifact it might be nice to trim it down.

Reply:
Are you talking about the bulk operations doc with the image if just an ID or my CMR operations one with a very tall mostly empty JSON box?

Reply:
Don't you _dare_ say 'yes.'